Fixed a typo of Umbraco! (#6893)
umbraco/UmbracoDocs: 15/umbraco-cms/implementation/custom-routing/signalR.md

## Define a custom route

> Next up, is defining a custom route. For this, we are going to use a `IAreaRoutes` and the base Umbraco backend path so we don't have to reserve another path in the settings.

Reviewer: ```suggestion
Next up, is defining a custom route. Use `IAreaRoutes` and the base Umbraco backend path so there's no need to reserve another path in the settings.
```
